tests/test_core.py

```diff
@@ -7256,7 +7260,8 @@ def setUp(self):
 binaryen3jo = make_run("binaryen3jo", compiler=CLANG, emcc_args=['-O3', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="interpret-binary,asmjs"'])
 
 # This only works when .emscripten specifies a JS_ENGINE with native wasm support.
-binaryen_native = make_run("binaryen_native", compiler=CLANG, emcc_args=['-O2', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"'])
+binaryen0_native = make_run("binaryen0_native", compiler=CLANG, emcc_args=['-O0', '-s', 'BINARYEN=1', '-s', 'BINARYEN_METHOD="native-wasm"'])
```

Comment:
at this point I think we don't need the interpreter modes, since we can finally test natively. how about making `binaryen0,1,2,3` test natively? shorter than writing `binaryen*_native`.

(we do need to test the binaryen interpreter, but the binaryen test suite does that)